### ISSUE TITLE
Publish new release please

### ISSUE THREAD

**Issue by barryib** [hypnoglow/helm-s3#153]
@hypnoglow thank you for your great work. I was wondering why can't you publish a new release of this plugin. There are lot of improvements in the aws-sdk-go added in the latest PR, but we can't use it easily beacause there is no release since Oct. 27 2020.

**Comment by georgefintechstudioscom** [comment]
I want to second @barryib on thanking @hypnoglow.  I also want to second his request for a new release soon.  We have found that Helm 3.4.0 has a bug that makes it impossible to use Helm 3 features like listing dependencies to local repos in charts, which was fixed in Helm 3.4.1 about 2 weeks after this plugin's last release (the support for Helm 3.5.0 in the unreleased version would cure that).  We're falling back to using Helm 2 charts in the meantime, but we've had to cut out our chart linter which is not as smart as Helm is about reading version 2 when needed.
